src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  let totalCost = days * dayCost;
+
+  if (days >= 3 && days < 7) {
+    totalCost = totalCost - 20;
+  }
+
+  if (days >= 7) {
+    totalCost = totalCost - 50;
+  }
+
+  return totalCost;
```

Comment:
The function name 'calculateRentalCost' is descriptive and good, but the constants for the daily cost, and the discount thresholds and amounts should be defined as constants at the top to avoid magic numbers in the code.